gusb/libgusb.ver

```diff
@@ -0,0 +1,42 @@
+LIBGUSB_0.1.0 {
+  global:
+    g_usb_*;
+  local: *;
+};
+
+LIBGUSB_0.1.1 {
+  global:
+    g_usb_device_get_platform_id;
+} LIBGUSB_0.1.0;
+
+LIBGUSB_0.1.5 {
+  global:
+    _g_usb_context_lookup_vendor;
+    _g_usb_context_lookup_product;
```

Comment:
those two are likely meant to be local - but I wasn't sure about them - hence the 'safe' route of setting them global; from what I gather, the old export script would have marked them internal too thouh